modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1911,7 +1910,16 @@ void setViewportLength(double value) {
         this.absoluteOffset = getPosition() * (estimatedSize - viewportLength);
         recalculateEstimatedSize();
     }
-    double getViewportLength() {
+
+    /**
+     * The length of the viewport portion of the {@code VirtualFlow} as computed during the layout pass.
```

Comment:
Suggested change:

"Returns the length of the viewport ..."

Reply:
Done